cypress/e2e/ConnectWallet.cy.ts

```diff
@@ -5,57 +5,83 @@ describe('Connect Wallet', () => {
 	});
 
 	afterEach(() => {
-		// reset connection for following test
+		// disconnect wallet for following test
 		cy.disconnectMetamaskWalletFromDapp().then((disconnected) => {
 			expect(disconnected).to.be.true;
 		});
 	});
 
 	after(() => {
-		cy.resetMetamaskAccount();
+		// reset account after test suite
+		cy.resetMetamaskAccount().then((reset) => {
+			expect(reset).to.be.true;
+		});
 	});
 
-	it('successfully opens the connect wallet modal', () => {
+	// fire click event on connect button
+	const clickConnectBtn = () =>
```

Comment:
As the `CONNECT` button is a css pseudo-element, we aren't able to use `cy.findByText()`. As a result, I've stuck with using Cypress' `cy.get()` and the class to find the element and click. Used `findBy...` wherever possible.

Reply:
Interesting! Whatever needs to be done 🫡